### DISCUSSION TITLE
Deletion of consumer notification

### DISCUSSION THREAD

**Discussion by chethan-b-hpe** [nats-io/nats.go#1748] [Q&A]
Is there any way that we can get to know about deletion of durable consumers due to inactivitythreshold which we set in consumer Config?

**Comment by Jarema** [comment]
you can check the advisories:

```
$JS.EVENT.ADVISORY.CONSUMER.DELETED.*.*
```

https://docs.nats.io/running-a-nats-service/nats_admin/monitoring/monitoring_jetstream

**Reply by chethan-b-hpe** [reply]
Let me give the context, so we are managing one safe queue to track consumers, so we want to delete from our safequeue once its get deleted due to this inactivity threshold, so for that is there any notifier that consumer can send when its deleted?
